config/application.rb

```diff
@@ -1,7 +1,6 @@
 require_relative 'boot'
 
 require 'rails/all'
-require  './app/middleware/middleware_healthcheck'
 
```

Comment:
I think we'll want to keep the middleware health check

Reply:
fixed